src/ScottPlot/Drawing/GDI.cs

```diff
@@ -95,6 +95,8 @@ public static System.Drawing.Graphics Graphics(Bitmap bmp, bool lowQuality = fal
         public static System.Drawing.Graphics Graphics(Bitmap bmp, PlotDimensions dims, bool lowQuality = false, bool clipToDataArea = true)
         {
             Graphics gfx = Graphics(bmp, lowQuality);
+            gfx.ScaleTransform(dims.TransformScale, dims.TransformScale);
```

Comment:
Very elegant, I wouldn't have thought to use a transform like this.

Reply:
Same! With all my exposure to System.Drawing I previously overlooked this method. Lots of good potential here

https://docs.microsoft.com/en-us/dotnet/api/system.drawing.graphics.scaletransform?view=net-5.0